expo/app.json

```diff
@@ -0,0 +1,33 @@
+{
+  "expo": {
```

Comment:
We have an `electric-sql` Expo account. The docs say:

> Add "owner": "electric-sql" under the "expo" key of app.json

https://docs.expo.dev/versions/latest/config/app/#owner